tests/qos/qos_sai_base.py

```diff
@@ -817,18 +817,36 @@ def updateIptables(
         for ipVersion in ipVersions:
             dut_asic.bgp_drop_rule(state="absent", **ipVersion)
 
-    @pytest.fixture(scope='class')
-    def stopServices(
+    @pytest.fixture(scope='class', name="stopServices")
+    def fixture_stopServices(
             self, duthosts, enum_rand_one_per_hwsku_frontend_hostname,
             enum_frontend_asic_index, swapSyncd, enable_container_autorestart,
-            disable_container_autorestart, get_mux_status, tbinfo, upper_tor_host,      # noqa F811
-            lower_tor_host, toggle_all_simulator_ports):                                # noqa F811
+            disable_container_autorestart, get_mux_status, tbinfo,
+            upper_tor_host, lower_tor_host, toggle_all_simulator_ports):
+        """
```

Comment:
wonder if this line should be removed?

Reply:
Removed. Missed it during copying over the code.